docs/userguide/mpi_apps.rst

```diff
@@ -347,4 +347,4 @@ batch job.
 
 #. MPI tasks may not span across nodes from more than one block.
 #. Parsl does not correctly determine the number of execution slots per block (`Issue #1647 <https://github.com/Parsl/parsl/issues/1647>`_)
-#. The executor uses a Python process per task, which can use a lot of memory (`Issue #2264 <https://github.com/Parsl/parsl/issues/2264>`_)
\ No newline at end of file
+#. The executor uses a Python process per task, which can use a lot of memory (`Issue #2264 <https://github.com/Parsl/parsl/issues/2264>`_)
```

Comment:
It looks like there's an unintentional whitespace change in this line. If this was not intentional, please remove the changes here.